pyproject.toml

```diff
@@ -162,9 +162,26 @@ source = ["scanpy"]
 omit = ["*/tests/*"]
 
 [tool.black]
-line-length = 88
-target-version = ["py38"]
 skip-string-normalization = true
-exclude = """
-/build/.*
-"""
+
+[tool.ruff]
```

Comment:
- no `exclude`, the default contains all we need
- no `line-length`, the default is 88
- don’t mention all the black-incompatible rules
- remove the “`F811` Redefinition of unused name” from per-file ignores, as it wasn’t triggered
- demote the “`E731` Do not assign a lambda expression” to a per-file ignore, as it’s only used there